service/history/workflow/update/errors_failures.go

```diff
@@ -43,4 +43,13 @@ var (
 			NonRetryable: true,
 		}},
 	}
+
+	acceptedUpdateCompletedWorkflowFailure = &failurepb.Failure{
+		Message: "Workflow Update is failed because it was accepted by Workflow but then Workflow completed.",
```

Comment:
Can I have a go at tweaking this error message a bit?

```suggestion
		Message: "Workflow Update failed because the Workflow completed before the Update completed.",
```

cc @drewhoskins-temporal